### DISCUSSION TITLE
Please reopen PR

### DISCUSSION THREAD

**Discussion by blut** [grafana/mimir#7149] [Help and support]
Please reopen PR: https://github.com/grafana/mimir/pull/4954
I've updated the code and tests as requested

**Comment by blut** [comment]
@dimitarvdimitrov @pracucci

**Comment by aknuds1** [comment]
https://github.com/grafana/mimir/pull/4954#issuecomment-1895525355

**Comment by blut** [comment]
Updated: https://github.com/grafana/mimir/pull/7150
